sql/2022/cms/cms_adoption_by_geo.sql

```diff
@@ -0,0 +1,52 @@
+#standardSQL
+# All CMS popularity per geo
+WITH geo_summary AS (
+  SELECT
+    `chrome-ux-report`.experimental.GET_COUNTRY(country_code) AS geo,
+    IF(device = 'desktop', 'desktop', 'mobile') AS client,
+    origin,
+    COUNT(DISTINCT origin) OVER (PARTITION BY country_code, IF(device = 'desktop', 'desktop', 'mobile')) AS total
+  FROM
+    `chrome-ux-report.materialized.country_summary`
+  WHERE
+    # We're intentionally using May 2022 CrUX data here.
+    # That's because there's a two month lag between CrUX and HA datasets.
+    # Since we're only JOINing with the CrUX dataset to see which URLs
+    # belong to different countries (as opposed to CWV field data)
+    # it's not necessary to look at the 202207 dataset.
+    yyyymm = 202205
```

Comment:
Should this be 202204 since we're a month earlier this year?

And same question to the other crux queries.

Reply:
Great point, thanks!